### PR TITLE
Added empty manuscript

### PR DESCRIPTION
**Pull request by SirReginald1**

### CODE REVIEW

File: src/stemmabench/algorithms/utils.py [hunk 1]
```diff
@@ -18,6 +19,19 @@ def load_text(path_to_text: str) -> str:
...
+        """For a given folder path returns a list of all the text names in that folder.
+        Will remove all names that contain the subsring "edge" from the list.
+        
+        ### Args:
+            - folder_path (str): The path to the folder that containing stemma texts.
+
+        ### Returns:
+            - list: List of manuscript names.
+        """
+        return [l.replace(".txt","") for l in listdir(folder_path) if l.find("edge") == -1]
```
Comment (SphRbtHyk) [inline src/stemmabench/algorithms/utils.py:33]: There a lot of nicer ways to do this, look into Pahtlib features.

For example : l.stem will give you the filename,  ... and you can glob from the start using .glob() (https://docs.python.org/3/library/pathlib.html):

```suggestion
        return [l.stem for l in Path(folder_path).glob("*.txt") if not "edge" in l.stem]
```

File: tests/test_manuscript_in_tree_empty.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        with self.assertRaises(ValueError, msg="Dose not throw error if children not a list."):
+            ManuscriptInTreeEmpty(label="label", children="test", parent=None)
+        with self.assertRaises(RuntimeError, msg="Dose not throw error if children and edges of different length."):
+            ManuscriptInTreeEmpty(label="label", children=[], parent=None, edges=[1])
+        with self.assertRaises(ValueError):
+             ManuscriptInTreeEmpty()
+        with self.assertRaises(ValueError, msg="The __init__ method does not raise a ValueError if recursive is specified and text_list is not."):
+             ManuscriptInTreeEmpty(parent=None, recursive={"label":{}}, children=[])
+
+    def test_Manuscript_attributes(self):
```
Comment (SphRbtHyk) [inline tests/test_manuscript_in_tree_empty.py:52]: majuscule in function name ;)

File: src/stemmabench/algorithms/utils.py [hunk 3]
```diff
@@ -18,6 +19,19 @@ def load_text(path_to_text: str) -> str:
         with open(path_to_text, "r", encoding="utf-8") as file:
             return file.read()
 
+    @staticmethod
+    def get_text_list(folder_path: str) -> List[str]:
+        """For a given folder path returns a list of all the text names in that folder.
+        Will remove all names that contain the subsring "edge" from the list.
+        
+        ### Args:
+            - folder_path (str): The path to the folder that containing stemma texts.
```
Comment (SphRbtHyk) [inline src/stemmabench/algorithms/utils.py:28]:
```suggestion
            - folder_path (str): The path to the folder that contains stemma texts.
```

File: src/stemmabench/algorithms/manuscript_in_tree_empty.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                 recursive: Union[Dict[str, dict], None] = None,
+                 text_list: Union[List[str], None] = None) -> None:
+        """A class representing the Manuscripts that make up the nodes of a stemma.
+
+        ### Args:
+            - parent (ManuscriptInTreeBase, Optional): The parent Manuscript of the curent Manuscript. If set to None should be the root of the tree.
+            - label (str, Optional): The label denoting this node.
+            - children (list, Optional): A list of this Manuscripts children.
+            - edges (list, Optional): A list representing the distance between the edges. Is in the same order as the list of children. 
+            - recursive (dict, Optional): Dictionary representation of the current Manuscript and all its decendents. If different than None will buil all the children of the manuscript
```
Comment (SphRbtHyk) [inline src/stemmabench/algorithms/manuscript_in_tree_empty.py:30]:
```suggestion
            - recursive (dict, Optional): Dictionary representation of the current Manuscript and all its decendents. If different than None will build all the children of the manuscript
```

File: src/stemmabench/algorithms/manuscript_in_tree_empty.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+            - from the given list. Should only be used when instantiating a stemma from the root.
+            - text_list (list, Optional): The list of texts present in the tree. Used with the recursive parameter, if current label
+            in recursive is not in text_list will instanciate an empty manuscript. Else will instantiate a manuscript. 
+
+        Raises:
+            - ValueError: If both recursive and lable are not specified.
+        """
+        if recursive:
+            if not text_list:
```
Comment (SphRbtHyk) [inline src/stemmabench/algorithms/manuscript_in_tree_empty.py:39]: encapsulate all of this logic within functions for easier debugging / testing

File: src/stemmabench/algorithms/manuscript_in_tree_empty.py (hunk 4 above)
Comment (SirReginald1) [reply src/stemmabench/algorithms/manuscript_in_tree_empty.py:30]: Done

File: src/stemmabench/algorithms/utils.py (hunk 3 above)
Comment (SirReginald1) [reply src/stemmabench/algorithms/utils.py:28]: Done

File: src/stemmabench/algorithms/utils.py (hunk 1 above)
Comment (SirReginald1) [reply src/stemmabench/algorithms/utils.py:33]: Done

File: tests/test_manuscript_in_tree_empty.py (hunk 2 above)
Comment (SirReginald1) [reply tests/test_manuscript_in_tree_empty.py:52]: Done

**Review by SphRbtHyk — APPROVED** [review]

**Pull request merged**